src/reviewing.md

```diff
@@ -0,0 +1,49 @@
+# rust-lang/rust review checklist
+
+- Does it have tests?
```

Comment:
These bullet points should be checklists, so they can be copy-pasted into comments and checked off.